.github/workflows/test.yml

```diff
@@ -46,5 +46,12 @@ jobs:
       - name: Install dependencies
         run: |
           uv pip install ".[optimizer,dev]"
+          uv pip install pytest pytest-cov
```

Comment:
I'd recommend getting rid of this line and adding `pytest-cov` to the `[dev]` dependencies in the `pyproject.toml` (`pytest` is also already there)